certif/app/adapters/session.js

```diff
@@ -59,4 +59,10 @@ export default class SessionAdapter extends ApplicationAdapter {
 
     return super.updateRecord(...arguments);
   }
+
+  dismissLiveAlert(sessionId, candidateId) {
+    const url = `${this.host}/${this.namespace}/sessions/${sessionId}/dismiss-live-alert`;
+    const payload = { data: { data: { attributes: { 'user-id': +candidateId } } } };
```

Comment:
Le `+` a un interet?
```suggestion
    const payload = { data: { data: { attributes: { 'user-id': candidateId } } } };
```